front/app/modules/commercial/moderation/hooks/useModerations.ts

```diff
@@ -65,21 +68,36 @@ export default function useModerations(props: InputProps) {
     setSearchTerm(searchTerm);
   }, []);
 
+  const onIsFlaggedChange = useCallback((isFlagged: boolean) => {
+    setIsFlagged(isFlagged);
+  }, []);
+
+  useEffect(() => {
```

Comment:
Tried making them more efficient like this, but removed them altogether in a different PR.

Reply:
Thanks for clarifying! 👍 
I won't repeat my comments for missing dependencies here in that case.